server/src/main/java/org/elasticsearch/transport/OutboundMessage.java

```diff
@@ -130,13 +128,6 @@ private static byte setStatus(boolean compress, boolean isHandshake, Writeable m
             super(version, setStatus(compress, isHandshake, message), requestId, message);
         }
 
-        @Override
-        public void writeTo(StreamOutput out) throws IOException {
-            if (version.before(Version.V_4_3_0)) {
-                out.setFeatures(Set.of());
```

Comment:
This would have never yield any operations because features are never transported anyway already before.